packages/website/lib/constants.js

```diff
@@ -1,5 +1,6 @@
 const API = /** @type {string} **/ (process.env.NEXT_PUBLIC_API);
 const MAGIC_TOKEN = /** @type {string} **/ (process.env.NEXT_PUBLIC_MAGIC);
+const MAGIC_TESTMODE_ENABLED = Boolean(process.env.NEXT_PUBLIC_MAGIC_TESTMODE_ENABLED);
```

Comment:
I think that we're relying on miniflare/wrangler to replace this `NEXT_PUBLIC_MAGIC_TESTMODE_ENABLED` is one reason you have to explicitly tell `miniflare` about your env vars either through the env file or through the `--binding KEY=VALUE` flags I added when the website ci workflow invokes `npm -w '@web3-storage/api' start`